src/test/java/api/support/fixtures/MaterialTypesFixture.java

```diff
@@ -42,6 +42,17 @@ public IndividualResource book()
     return materialTypeRecordCreator.createIfAbsent(book);
   }
 
+  public IndividualResource any()
```

Comment:
Can you help me understand what this represents, how can there be an `any` material type?

Reply:
I removed it.  I missed it when cleaning up the code before committing.